src/System.Linq/src/System/Linq/ElementAt.cs

```diff
@@ -64,7 +35,8 @@ public static TSource ElementAtOrDefault<TSource>(this IEnumerable<TSource> sour
             if (partition != null)
             {
                 bool found;
-                return partition.TryGetElementAt(index, out found);
+                element = partition.TryGetElementAt(index, out found);
```

Comment:
I should say so. The rather unusual way round they are is an artefact of the incremental way they were arrived at rather than for any other reason (I tried changing before and it had a tiny negative impact on performance, but I wouldn't worry about it if we'd a better reason for doing the change as we do now, and also it's likely going to be a small improvement now that it matches more code at the public interface).